partcad/src/partcad/wrappers/wrapper_obj.py

```
@@ -0,0 +1,66 @@
+
+        for face in faces:
+            polygon = BRepBuilderAPI_MakePolygon()
+            for vertex_idx in face:
+                x, y, z = vertices[vertex_idx - 1]
+                polygon.Add(gp_Pnt(x, y, z))
+            polygon.Close()
+
+            face_shape = BRepBuilderAPI_MakeFace(polygon.Wire()).Face()
+            builder.Add(compound, face_shape)
+
+    except Exception as e:
```

Ruff (0.8.2)

Do not catch blind exception: `Exception`

(BLE001)